lib/school_mgt/students.ex

```diff
@@ -16,10 +16,12 @@ defmodule SchoolMgt.Students do
   end
 
   def get_student(id) do
-    Repo.get!(Student, id)
+    sid = String.to_integer(id)
```

Comment:
i am sure we dont need to use String.to_integer() , please check it again

Reply:
done
it works 
but it didn't work for at that time so I did that